frontend/src/APIClients/BaseGQLClient.ts

```diff
@@ -17,10 +17,10 @@ import {
 } from "../utils/LocalStorageUtils";
 import { REFRESH } from "./mutations/AuthMutations";
 
-const httpLink = createHttpLink({
+const httpLink = createUploadLink({
   uri: `${process.env.REACT_APP_BACKEND_URL}/graphql`,
   credentials: "include",
-});
+}) as unknown as ApolloLink;
```

Comment:
Need `as unknown` because it seems that the `link` types differ slightly between the `@apollo/client` and `apollo-upload-client` versions